src/httprpc.cpp

```diff
@@ -199,10 +202,13 @@ static bool HTTPReq_JSONRPC(const std::any& context, HTTPRequest* req)
                 req->WriteReply(HTTP_FORBIDDEN);
                 return false;
             }
-            UniValue result = tableRPC.execute(jreq);
 
-            // Send reply
-            strReply = JSONRPCReply(result, NullUniValue, jreq.id);
+            jreq.parse(valRequest);
+            if (jreq.IsNotification()) {
+                req->WriteReply(HTTP_NO_CONTENT);
+                return true;
```

Comment:
Shouldn't we still execute the notification? While it's not possible for the client to confirm it worked, it should still execute...

Reply:
My thinking was that since our RPC API does not have any notifications defined yet we could discard them for now. Like, our RPCs and responses are a separate category from notifications.  But you have an interesting point, a user could send a notification with a method like `generatetoaddress` and just not want to wait for a reply. There would be plenty of "getter" RPCs that would make no sense in a notification however and just waste time on the server. Maybe this is best suited for a follow-up PR that establishes a notification interface for bitcoind and specifies which commands are allowed there?

Reply:
Yep good call after doing some more reading about this in other implementations, I updated this branch to match and tested it. So now notifications are executed but send no response.